os/os.proto

```diff
@@ -65,14 +67,22 @@ service OS {
   //                              <-- [Validated|InstallError]
   //
   // On a dual Supervisor Target, only the Active Supervisor runs this gNOI
-  // Service. The Install RPC applies to the Active Supervisor unless
+  // Service. 
+  // On systems which support installing the entire system with
+  // one upgrade command, one install RPC is sufficient to install the package 
+  // on all Supervisors. 
+  // On other systems, the Install RPC applies to the Active Supervisor unless
   // InstallRequest->TransferRequest->standby_supervisor is set, in which case
   // it applies to the Standby Supervisor. One Install RPC is required for each
   // Supervisor. The Supervisor order of package installation MUST not be fixed.
   //
-  // The Target MUST always attempt to copy the OS package between Supervisors
-  // first before accepting the transfer from the Client. The syncing progress
-  // is reported to the client with InstallResponse->SyncProgress messages.
+  // On systems which support installing the entire system with one upgrade
+  // command, package is available on all Supervisors or none. If the package
+  // is absent on the Active Supervisor, Target does not try to copy the package
+  // from the standby Supervisor.
+  // On other systems, the Target MUST always attempt to copy the OS package 
+  // between Supervisors first before accepting the transfer from the Client. The 
+  // syncing progress is reported to the client with InstallResponse->SyncProgress messages.
```

Comment:
On a dual Supervisor Target which requires installing the entire system with one Install RPC, one Install RPC is sufficient to install the package on all Supervisors. 

On a dual Supervisor Target which requires one Install RPC per supervisor, the Install RPC applies to the Active Supervisor unless InstallRequest->TransferRequest->standby_supervisor is set, in which case it applies to the Standby Supervisor. The Target MUST accept any order of package installation.

On a dual Supervisor Target which requires installing the entire system with one Install RPC, the package MUST be made available on all Supervisors or none. If upon installation the package is absent from either Supervisor, the Target will transfer to both Supervisors overwriting the existing one.

On a dual Supervisor Target which requires one Install RPC per supervisor, the Target MUST always attempt to copy the OS package between Supervisors first before accepting the transfer from the Client. The syncing progress is reported to the client with InstallResponse->SyncProgress messages.

Reply:
done